### PR TITLE
FAODEL: Add version 1.1906.1

### PR DESCRIPTION
**Pull request by tkordenbrock**
Update the FAODEL package with version 1.1906.1.

### CODE REVIEW

File: var/spack/repos/builtin/packages/faodel/package.py [hunk 1]
```diff
@@ -33,6 +34,8 @@ class Faodel(CMakePackage):
     variant('network', default='nnti', values=('nnti', 'libfabric'),
             description='RDMA Network library to use for \
                          low-level communication')
+    variant('cereal', default=False,
+            description='Use Cereal to serialize NNTI data structures else XDR')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/faodel/package.py]:
```
var/spack/repos/builtin/packages/faodel/package.py:38: [E501] line too long (80 > 79 characters)
```
I know it seems counterintuitive, but if you put the entire variant on a single line, Spack will ignore it in the flake8 line length checks.

Comment (tkordenbrock) [reply var/spack/repos/builtin/packages/faodel/package.py]: @adamjstewart Thanks for the Pro Tip.  Is it preferred to put the entire variant on one line?  I looked at other packages and it looks like it goes both ways.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/faodel/package.py]: No preference. I usually put everything on one line, other contributors usually wrap things. As long as flake8 is happy, no one will complain.

Comment (tkordenbrock) [reply var/spack/repos/builtin/packages/faodel/package.py]: OK.  I just pushed a new commit with each variant on a single line.

**Pull request merged**